FreeRTOS-Plus/Test/FreeRTOS_Plus_TCP_Integration_Tests/WinPCap/Packet32.h

```diff
@@ -0,0 +1,359 @@
+/*
```

Comment:
Is it really needed to introduce ```WinPCap``` to this test project? Would it be possible to mock a network connection, since the functions tested in this PR seems not relevant to the actual connection?

I'm not sure whether this project will later include connection related tests. But it seems to me, we could even skip all the task setup related (scheduler, task creation etc.) and only test the parsing logic.... 

(I'm not sure about the right answer, asking out of genuine curiosity. And also asking to get concise and to-the-point tests in our repo.)